www/src/components/sidebar/item.js

```diff
@@ -14,6 +14,19 @@ const isItemActive = (activeItemParents, item) => {
 }
 
 class Item extends React.PureComponent {
+  constructor(props, context) {
+    super(props, context)
+    this.itemRef = React.createRef()
```

Comment:
I would name it `activeItemRef` to make clear it is not on every item

Reply:
Agreed, done.